data/json/items/tool/metalworking.json

```diff
@@ -3,35 +3,65 @@
     "id": "anvil",
     "type": "TOOL",
     "name": { "str": "anvil" },
-    "description": "An enormously heavy block of oddly-shaped steel with a chisel-like projection set into the corner.  It's used in most metalworking fabrication recipes.",
-    "weight": "54000 g",
-    "volume": "24 L",
+    "description": "A heavy block of oddly-shaped steel complete with a pritchel hole and hardy hole.  Once installed it can be used in most metalworking fabrication recipes.",
+    "weight": "26000 g",
+    "volume": "12 L",
     "price": 100000,
     "price_postapoc": 2000,
     "to_hit": -5,
     "material": [ "steel" ],
     "symbol": ";",
     "color": "dark_gray",
-    "qualities": [ [ "ANVIL", 3 ] ],
     "flags": [ "DURABLE_MELEE" ],
     "melee_damage": { "bash": 40 }
   },
   {
     "id": "anvil_bronze",
     "type": "TOOL",
     "name": { "str": "bronze anvil" },
-    "description": "A block of oddly-shaped bronze with a chisel-like projection set into the corner.  It's used in most metalworking fabrication recipes.",
-    "weight": "2180 g",
-    "volume": "2500 ml",
+    "description": "A block of oddly-shaped bronze complete with a pritchel hole and hardy hole.  Once installed it can be used in most metalworking fabrication recipes.",
```

Comment:
```suggestion
    "description": "A block of oddly-shaped bronze complete with a pritchel hole and hardy hole.  Once installed, it can be used in most metalworking fabrication recipes.",
```